doc/api/addons.md

```diff
@@ -213,6 +213,24 @@ Addon developers are recommended to use to keep compatibility between past and
 future releases of V8 and Node.js. See the `nan` [examples][] for an
 illustration of how it can be used.
 
+
+## N-API
+
+N-API is a newer API for building native Addons.  It is independent from
+the underlying JavaScript runtime (ex v8) and is maintained as part of
+Node.js itself.  This API will be Application Binary Interface (ABI) stable
+across version of Node.js.  It is intended to insulate Addons from
+changes in the underlying JavaScript engine and allow modules
+compiled for one version to run on later versions of Node.js without
+recompilation.  Addons are built/packaged with the same approach/tools
+outlined in this document(node-gyp, etc.). The only difference is the
+set of APIs that are used by the native code.  Instead of using the V8
+or [Native Abstractions for Node.js][] APIs, the functions available
+in the N-API are used instead.
```

Comment:
This sentence begins and ends with "instead." I think the second one can be dropped.